gcp/vars.tf

```diff
@@ -46,6 +46,6 @@ variable "package_url" {
 
 variable "admin_server_instance_type" {
   type        = string
-  description = "URL of the Project N package to install on launch"
-  default = "n1-standard-1"
+  description = "The admin server's instance type"
+  default     = "n2-standard-1"
```

Comment:
do we need to specify this in 2 separate places?

Reply:
Yes.